megamek/src/megamek/client/ui/swing/UnitEditorDialog.java

```diff
@@ -1177,12 +1229,51 @@ private void btnOkayActionPerformed(java.awt.event.ActionEvent evt) {
                 aero.setLeftThrustHits(leftThrusterCrit.getHits());
             }
             if (null != rightThrusterCrit) {
-                aero.setLeftThrustHits(leftThrusterCrit.getHits());
+                aero.setRightThrustHits(rightThrusterCrit.getHits());
             }
             if ((null != dockCollarCrit) && (aero instanceof Dropship)) {
                 ((Dropship) aero)
                         .setDamageDockCollar(dockCollarCrit.getHits() > 0);
             }
+            if ((null != kfboomCrit) && (aero instanceof Dropship)) {
```

Comment:
Does KF Boom damage only apply to dropships?  Looks like it can't be applied to small craft of jumpships, just wanted to make sure.

Reply:
Yes. Dropships only. Jumpships can't carry each other, so they have no boom and small craft fit in bays and don't need one.   If we ever get to modular space stations from IO, they'll need a KF-Boom too and I can adjust this.